backend/functions/case-associated/case-associated.function.ts

```diff
@@ -14,11 +14,18 @@ const httpTrigger: AzureFunction = async function (
   functionContext: Context,
   request: HttpRequest,
 ): Promise<void> {
-  const applicationContext = await applicationContextCreator(functionContext);
+  const applicationContext = await ContextCreator.applicationContextCreator(
+    functionContext,
+    request,
+  );
   const controller = new CaseAssociatedController(applicationContext);
+
   try {
+    applicationContext.session =
+      await ContextCreator.getApplicationContextSession(applicationContext);
+
     const responseBody = await controller.getAssociatedCases(applicationContext, {
```

Comment:
I would love to see the controllers get parameters they need from the context and remove this kind of logic from the Azure functions to minimize the platform specific code.

Not for this PR or ticket.